src/adapter/dto/post_dto.py

```
@@ -0,0 +1,39 @@
+from core.domain import Post,Category
+from datetime import datetime
+from infrastructure.sqlalchemy.model import Post
```

Ruff (0.8.2)

Redefinition of unused `Post` from line 1

(F811)